packages/wrangler/src/utils/compatibility-date.ts

```diff
@@ -13,15 +23,25 @@ export function getDevCompatibilityDate(
 	const miniflareWorkerd = miniflareRequire("workerd") as {
```

Comment:
I think we now also have a dependency from Wrangler directly on workerd, so we could probably simplify this code to just get that directly.

Reply:
This PR only ~moves code around, we can handle that in a follow up PR maybe?

Reply:
Sure.